src/jsep.js

```diff
@@ -551,6 +587,16 @@
 					};
 				},
 
+				expression_parser_scope = {
+					get index() { return index; },
+					set index(v) { index = v; },
+					exprI: exprI,
```

Comment:
Might be useful to include all methods here? Too bad it feels like code duplication though...